### PR TITLE
port to 1.21 + cleanup

### PR DESCRIPTION
**Pull request by BuggyAl**
I haven't done extensive testing but it compiles and runs properly. I also ran the IntelliJ reformatter to clean up some things in the code.

### CODE REVIEW

**Comment by BuggyAl** [comment]
Link to the compiled jar for those who want it: https://github.com/BuggyAl/visiblebarriers/releases/tag/v2.0.14

File: src/main/java/xyz/amymialee/visiblebarriers/mixin/EntityTypeMixin.java
Comment (AmyMialee) [inline src/main/java/xyz/amymialee/visiblebarriers/mixin/EntityTypeMixin.java:1]: Generally I prefer keeping the shadow fields more compacted and out of the way, but this is just preference.

File: src/main/java/xyz/amymialee/visiblebarriers/mixin/boxing/AbstractBlockMixin.java
Comment (AmyMialee) [inline src/main/java/xyz/amymialee/visiblebarriers/mixin/boxing/AbstractBlockMixin.java:1]: I do also tend to prefer keeping empty methods on one line where possible, but this is fine.

**Review by AmyMialee — APPROVED** [review]
Thanks for the help!

**Pull request merged**